src/pages/candidate-info/candidate-info.html

```diff
@@ -1,86 +1,119 @@
 <ion-header>
-    <page-header></page-header>
+    <mes-header [title]='getTitle()'></mes-header>  
 </ion-header>
 
 <ion-content padding class="mes-main-content">
+    <div class="mes-header-xlg mes-view-candidate-header-padding-fix">
+        <ion-row>
+            <ion-col col-9 text-left>
+                {{getCandidateName()}}
+            </ion-col>
+            <ion-col col-3 text-right>
+                {{slotDetail.startTime}}
+            </ion-col>
+        </ion-row>
+    </div>
+    <ion-card class="mes-card">
+        <ion-card-header class="mes-header-lg">
+            Candidate information 
+        </ion-card-header>
+        <ion-card-content>
+            <ion-row>
+                <ion-col col-4 class="mes-bold">
+                    Test category
+                </ion-col>
+                <ion-col col-8>
+                    Category B
+                </ion-col>
+            </ion-row>
+            <div class="mes-divider"></div>
+            <ion-row>
+                <ion-col col-4>
+                    Slot
+                </ion-col>
+                <ion-col col-8>
+                    {{getSLotType()}}
```

Comment:
getSlotType - might be good to write a lowercase L here.

Reply:
fixed 👍